### PR TITLE
TOOL-11245 leverage linux-pkg to build delphix-virtualization package

### PR DESCRIPTION
**Pull request by prakashsurya**

### CODE REVIEW

**Comment by prakashsurya** [comment]
build-package is [here](http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/linux-pkg/job/custom/job/build-package/job/custom/29/console)

File: packages/virtualization/config.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		"$WORKDIR/artifacts"
+	logmust cp -v \
+		packaging/build/reports/dependency-license/* \
+		"$WORKDIR/artifacts/"
+
+	#
+	# Build the "toolkit-devkit" artifacts
+	#
+	logmust cd "$WORKDIR/repo/appliance/toolkit"
+	# TODO: set version.number correctly
```
Comment (prakashsurya) [inline packages/virtualization/config.sh:94]: TODO: set version.number correctly

File: packages/virtualization/config.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+# shellcheck disable=SC2034
+
+DEFAULT_PACKAGE_GIT_URL="https://gitlab.delphix.com/app/dlpx-app-gate.git"
+PACKAGE_DEPENDENCIES="adoptopenjdk crypt-blowfish misc-debs"
+
+function prepare() {
+	logmust install_pkgs \
+		"$DEPDIR"/adoptopenjdk/*.deb \
+		"$DEPDIR"/crypt-blowfish/*.deb \
+		"$DEPDIR"/misc-debs/unzip_6.0-21ubuntu1_amd64.deb
```
Comment (sebroy) [inline packages/virtualization/config.sh:26]: Question: Seems weird that declaring `PACKAGE_DEPENDENCIES` doesn't automatically do this for you. Do you know why that is?

**Review by sebroy — APPROVED** [review]

File: packages/virtualization/config.sh (hunk 2 above)
Comment (prakashsurya) [reply packages/virtualization/config.sh:26]: That'd be a question for @pzakha. As far as I know, this is how linux-pkg works.

**Review by matt-skinner-delphix — APPROVED** [review]

**Pull request merged**